java.lang.NumberFormatException: For input string: "0.0044"

Bug in version 2.0.0-CI.SNAPSHOT (from git)

Asterisk 13.1.0 built by root @ sip on a x86_64 running Linux

```
ERROR 2015-05-05 13:35:47,488 [Asterisk-Java ManagerConnection-0-Reader-0] (Slf4JLogger.java:134) - Unable to set property 'rtt' to '0.0044' on org.asteriskjava.manager.event.RtcpReceivedEvent
java.lang.reflect.InvocationTargetException
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.asteriskjava.manager.internal.AbstractBuilder.setAttributes(AbstractBuilder.java:136)
	at org.asteriskjava.manager.internal.EventBuilderImpl.buildEvent(EventBuilderImpl.java:411)
	at org.asteriskjava.manager.internal.ManagerReaderImpl.buildEvent(ManagerReaderImpl.java:348)
	at org.asteriskjava.manager.internal.ManagerReaderImpl.run(ManagerReaderImpl.java:224)
	at java.lang.Thread.run(Thread.java:745)
Caused by: java.lang.NumberFormatException: For input string: "0.0044"
	at java.lang.NumberFormatException.forInputString(NumberFormatException.java:65)
	at java.lang.Long.parseLong(Long.java:441)
	at java.lang.Long.parseLong(Long.java:483)
	at org.asteriskjava.manager.event.AbstractRtcpEvent.secStringToLong(AbstractRtcpEvent.java:101)
	at org.asteriskjava.manager.event.RtcpReceivedEvent.setRtt(RtcpReceivedEvent.java:202)
	... 9 more
```

--- a/src/test/java/org/asteriskjava/manager/event/RtcpReceivedEventTest.java
+++ b/src/test/java/org/asteriskjava/manager/event/RtcpReceivedEventTest.java
@@ -49,6 +49,6 @@ public class RtcpReceivedEventTest
     public void testRtt()
     {
         rtcpReceivedEvent.setRtt("12345(sec)");
-        assertEquals(new Long(12345), rtcpReceivedEvent.getRtt());
+        assertEquals(new Double(12345), rtcpReceivedEvent.getRtt());
     }
 }

